### ISSUE TITLE
Errors in the station "Schlossplatz" 

### ISSUE THREAD

**Issue by sara-georgy** [stadtnavi/digitransit-ui#688]
1. 4 out of 6 stations "Schlossplatz" show the message "No departures"

![image](https://user-images.githubusercontent.com/84382750/122970051-aced1480-d38d-11eb-98b3-96e6024cfe4b.png)

2. The only two stations that show departures have another error when clicking on the tab "timetable"

![image](https://user-images.githubusercontent.com/84382750/122970222-eaea3880-d38d-11eb-868a-816b3839412d.png)

![image](https://user-images.githubusercontent.com/84382750/122970276-fc334500-d38d-11eb-861d-01205483d6bc.png)

![image](https://user-images.githubusercontent.com/84382750/122970323-0a816100-d38e-11eb-8f18-28a30e31362a.png)

**Comment by MartinH-open** [comment]
Same kind of issue here:  
(here follows English translation of what I submitted as feedback in the StadtNavi form.)
As an example, what is difficult with the stops. In Nufringen there are three bus stops in one place. 
The two stops NUFRINGEN KUPPINGER STR. seem to be meaningfully defined for the two directions. 
The third alleged bus stop Nufringen is more of a dummy. There are now no departures and the timetable says for the day: Today nothing about "Departures by hour (minutes / route)" (it says so in the German version). And there are no disturbances either. This entry is therefore completely worthless and should not be displayed in the first place.

![image](https://user-images.githubusercontent.com/1383111/123011025-fc4c3880-d3bf-11eb-9471-d6a8974571e8.png)

**Comment by leonardehrenfried** [comment]
Well, not sure when the screenshots were taken (1AM?) but often there just isn't a departure coming up, articularly at the smaller stops.

What happens when you select the timetable tab?

**Comment by sara-georgy** [comment]
My screenshots were taken around 6pm. 

I checked the stations again now and they are still empty. The timetable tab shows the following:
![image](https://user-images.githubusercontent.com/84382750/123064001-2fb5b400-d40e-11eb-81da-c2d2bd5a996f.png)

If there isn't a timetable provided or any departures coming up, shouldn't we take the empty stations out of the map?
I think it is so confusing that there are multiple duplicates of one station, and 4 out of 6 are empty.

**Comment by leonardehrenfried** [comment]
Yes, agreed. I personally like the way that Helsinki have grouped their bus stops: https://reittiopas.hsl.fi/terminaalit/HSL%3A1000002

You can click on both the "terminal" (large bus icon) and the individual stops.

On the other hand the UX there is confusing, too, as icons appear and disappear seemingly at random. :man_shrugging:

**Comment by hbruch** [comment]
The stop in Nufringen is a rail replacement stop which is served on weekends, presumably when there are constructions works.
At the Schlossplatz, I expect that to be night bus stops only running on the weekend evenings.

Grouping, like Helsinki does, is nice, but they only do so for train/bus stations, not every smaller stop, no?

In the case of Nufringen, that might probably not even work, as the "Nufringen" stop is assigned to the Nufringen S-Bahn-Station:

```
$ grep Nufringen stops.txt
…
"de:08115:4736:0:3","Nufringen Kuppinger Str.","48.6204217697254","8.88721257747833"
"de:08115:4736:0:4","Nufringen Kuppinger Str.","48.6205476660928","8.8872122984599"
…
"de:08115:5775:1:1","Nufringen","48.6201544476336","8.88976313287869"
"de:08115:5775:1:2","Nufringen","48.6200826229923","8.88988536096008"
„de:08115:5775:7:5","Nufringen","48.6204936708599","8.88717172695626"
…
$ grep de:08115:5775:7:5 stop_times.txt
...
"47.T3.31-1-u-j21-11.1.H","20:30:00","20:30:00","de:08115:5775:7:5","2","","0","0",“5007.37"
...
$ grep 47.T3.31-1-u-j21-11.1.H trips.txt
"31-1-u-j21-11","T3#4","47.T3.31-1-u-j21-11.1.H","31-1-u-j21-11.1.H","Herrenb. ZOB Bahnhofstraße","0",""
$ grep 31-1-u-j21-11 routes.txt
"31-1-u-j21-11","","S1E","S1E","3","FF0000","FFFFFF"
$ grep "T3#4" calendar.txt
"T3#4","0","0","0","0","0","0","1","20210731","20210808"
"T3#40","0","0","0","0","0","0","1","20210625",“20210627"
```
